quarkusQS/pom.xml

```diff
@@ -0,0 +1,191 @@
+            <goals>
+              <goal>deploy</goal>
+            </goals>
+          </execution>
+         <!-- undeploy doesnt exist -->
+        </executions>
+
+        <configuration>
+          <auth>
+            <type>service_principal</type>
+            <client>${azure.clientId}</client>
+            <tenant>${azure.tenantId}</tenant>
+            <key>${azure.password}</key>
+            <environment>azure</environment>
+          </auth>
+          <schemaVersion>v2</schemaVersion>
+          <subscriptionId>${azure.subscriptionId}</subscriptionId>
+          <resourceGroup>${azure.group}</resourceGroup>
+          <appName>${azure.appName}</appName>
+          <pricingTier>P1v2</pricingTier>
+          <region>eastus2</region>
+
+          <runtime>
+            <os>Linux</os>
+            <javaVersion>Java 11</javaVersion>
+            <webContainer>Java SE</webContainer>
+          </runtime>
+          <appSettings>
+            <property>
+              <name>PORT</name>
+              <value>8080</value>
+            </property>
+            <property>
+              <name>WEBSITES_PORT</name>
+              <value>8080</value>
+            </property>
+            <property>
+              <name>WEBSITES_CONTAINER_START_TIME_LIMIT</name>
+              <value>600</value>
+            </property>
+          </appSettings>
+          <deployment>
+            <resources>
+              <resource>
+                <directory>${project.basedir}/target</directory>
+                <includes>
+                  <include>*.jar</include>
+                </includes>
+              </resource>
+            </resources>
+          </deployment>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>${quarkus.platform.group-id}</groupId>
+        <artifactId>quarkus-maven-plugin</artifactId>
+        <version>${quarkus.platform.version}</version>
+        <extensions>true</extensions>
+        <executions>
+          <execution>
+            <goals>
+              <goal>build</goal>
+              <goal>generate-code</goal>
+              <goal>generate-code-tests</goal>
+            </goals>
+          </execution>
+        </executions>
+
+      </plugin>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>${compiler-plugin.version}</version>
+        <configuration>
+          <compilerArgs>
+            <arg>-parameters</arg>
+          </compilerArgs>
+        </configuration>
+      </plugin>
+      <plugin>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <version>${surefire-plugin.version}</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>integration-test</goal>
+              <goal>verify</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <systemPropertyVariables>
+            <azure.appName>${azure.appName}</azure.appName>
```

Comment:
Why do you set only two values? I would expect all azure.* properties or none. Are the other properties automatically propagated?

I am not saying this is wrong, I am just confused.

Reply:
other azure properties are used by azure plugin for authentication (in different mvn phase), but only these two need to be propagated to `QuarkusTestConstants`

Reply:
Doesn't it work without this? You have `sunstone.azure.password=${azure.password}` in sunstone.properties and it seems to work without specifying it in here.

Reply:
it could work w/o but I want to propagate unique name/timestamp this way from pom

Reply:
ok